istio-pilot-discovery-1.20.yaml

```diff
@@ -29,6 +29,7 @@ pipeline:
     with:
       packages: ./pilot/cmd/pilot-discovery
       output: pilot-discovery
+      deps: github.com/go-jose/go-jose/v3@v3.0.1
```

Comment:
```suggestion
```

Reply:
can we switch to using go bump 

```
  - uses: go/bump
    with:
      deps: github.com/go-jose/go-jose/v3@v3.0.1
      go-version: "1.21"
 ```

Reply:
Amazing, done! I didn't realize it was already live 😍